### PR TITLE
Fixed encoding error on the tuna/__init__.py file

### PR DESCRIPTION
**Pull request by b1quint**

### CODE REVIEW

**Comment by rcbrgs** [comment]
Hi Bruno!
I think this change is fine, and that the integration test fails for a separate reason.
If you check the Travis output, it uses Python 3.3.5, but Tuna tries to install (from pip_requirements.txt) matplotlib, of which the version being installed (2.0.0) requires Python 3.4 or later.
The way to fix this is by specifying on pip_requirements.txt that matplotlib should have a specific version, probably the latter version in the 1.x series, that can run under Python 3.3.5.
This would (probably) solve the integration test issue, which happened noz not because of your PR specifically, but because Tuna has not been tested for a long time, and a long time ago matplotlib 1.x was the default.
In other words, the default for matplotlib doesn't work anymore and this breaks tests. I would like to fix tests before merging anything into the repository.

**Comment by rcbrgs** [comment]
Ok, I fixed everything in the branch "next", which is where we commit new things. I will close this PR so you can submit a PR to the "next" branch. It should pass tests and then I will merge it.

**Comment by b1quint** [comment]
Perfect! Thanks!
